cross/squid/PLIST

```diff
@@ -1,75 +1,36 @@
+bin:libexec/basic_nis_auth
+bin:libexec/basic_pam_auth
+rsc:libexec/basic_pop3_auth
+bin:libexec/basic_radius_auth
+bin:libexec/basic_smb_auth
+rsc:libexec/basic_smb_auth.sh
+rsc:libexec/basic_smb_lm_auth
 rsc:libexec/cachemgr.cgi
-bin:libexec/digest_pw_auth
+rsc:libexec/cert_valid.pl
+bin:libexec/digest_file_auth
+bin:libexec/digest_ldap_auth
 bin:libexec/diskd
-bin:libexec/fakeauth_auth
-bin:libexec/getpwname_auth
-bin:libexec/msnt_auth
-bin:libexec/ncsa_auth
-rsc:libexec/no_check.pl
+rsc:libexec/helper-mux.pl
+rsc:libexec/log_db_daemon
+rsc:libexec/log_file_daemon
+bin:libexec/negotiate_wrapper_auth
+bin:libexec/ntlm_fake_auth
 bin:libexec/ntlm_smb_lm_auth
-rsc:libexec/pop3.pl
-bin:libexec/smb_auth
-rsc:libexec/smb_auth.pl
-rsc:libexec/smb_auth.sh
-rsc:libexec/squid_db_auth
-bin:libexec/squid_radius_auth
-bin:libexec/squid_unix_group
+bin:libexec/ssl_crtd
+rsc:libexec/storeid_file_rewrite
 bin:libexec/unlinkd
-rsc:libexec/wbinfo_group.pl
-bin:libexec/yp_auth
-rsc:share/errors/templates/ERR_ACCESS_DENIED
-rsc:share/errors/templates/ERR_ACL_TIME_QUOTA_EXCEEDED
-rsc:share/errors/templates/ERR_AGENT_CONFIGURE
-rsc:share/errors/templates/ERR_AGENT_WPAD
-rsc:share/errors/templates/ERR_CACHE_ACCESS_DENIED
-rsc:share/errors/templates/ERR_CACHE_MGR_ACCESS_DENIED
-rsc:share/errors/templates/ERR_CANNOT_FORWARD
-rsc:share/errors/templates/ERR_CONNECT_FAIL
-rsc:share/errors/templates/ERR_DIR_LISTING
-rsc:share/errors/templates/ERR_DNS_FAIL
-rsc:share/errors/templates/ERR_ESI
-rsc:share/errors/templates/ERR_FORWARDING_DENIED
-rsc:share/errors/templates/ERR_FTP_DISABLED
-rsc:share/errors/templates/ERR_FTP_FAILURE
-rsc:share/errors/templates/ERR_FTP_FORBIDDEN
-rsc:share/errors/templates/ERR_FTP_NOT_FOUND
-rsc:share/errors/templates/ERR_FTP_PUT_CREATED
-rsc:share/errors/templates/ERR_FTP_PUT_ERROR
-rsc:share/errors/templates/ERR_FTP_PUT_MODIFIED
-rsc:share/errors/templates/ERR_FTP_UNAVAILABLE
-rsc:share/errors/templates/ERR_GATEWAY_FAILURE
-rsc:share/errors/templates/ERR_ICAP_FAILURE
-rsc:share/errors/templates/ERR_INVALID_REQ
-rsc:share/errors/templates/ERR_INVALID_RESP
-rsc:share/errors/templates/ERR_INVALID_URL
-rsc:share/errors/templates/ERR_LIFETIME_EXP
-rsc:share/errors/templates/ERR_NO_RELAY
-rsc:share/errors/templates/ERR_ONLY_IF_CACHED_MISS
-rsc:share/errors/templates/error-details.txt
-rsc:share/errors/templates/ERR_PRECONDITION_FAILED
-rsc:share/errors/templates/ERR_READ_ERROR
-rsc:share/errors/templates/ERR_READ_TIMEOUT
-rsc:share/errors/templates/ERR_SECURE_CONNECT_FAIL
-rsc:share/errors/templates/ERR_SHUTTING_DOWN
-rsc:share/errors/templates/ERR_SOCKET_FAILURE
-rsc:share/errors/templates/ERR_TOO_BIG
-rsc:share/errors/templates/ERR_UNSUP_HTTPVERSION
-rsc:share/errors/templates/ERR_UNSUP_REQ
-rsc:share/errors/templates/ERR_URN_RESOLVE
-rsc:share/errors/templates/ERR_WRITE_ERROR
-rsc:share/errors/templates/ERR_ZERO_SIZE_OBJECT
+bin:libexec/url_fake_rewrite
+rsc:libexec/url_fake_rewrite.sh
+bin:sbin/squid
+rsc:share/errors/COPYRIGHT
```

Comment:
Combine with `rsc:share/errors`, that way the PLIST file can get a whole lot smaller.